### DISCUSSION TITLE
Saving CSV files using PHP

### DISCUSSION THREAD

**Discussion by ghost** [jspsych/jsPsych#2480] [Q&A]
Hello,
I am having real difficulties saving csv files to my server.
I have followed the instructions outlined on this page, and my php file is in the same folder as my experiment on the server :
https://softdev.ppls.ed.ac.uk/online_experiments/example_code/datasaving.html#datasaving
My code for the php file and the saving data code is the same as the example here so I am not sure what I am doing wrong. No csv file appears at all when I have tried implementing this code. Would greatly appreciate any help
Thank you

**Comment by jodeleeuw** [comment]
Hi @Vizitok 

Are you using jsPsych 6.3 like they are in the example?

**Comment by ghost** [comment]
No I am using jspsych 7.0, but I added the equivalent plugins in jspsych 7 i.e. keyboard-html plugin
do you have to use a different method for saving using phps in jspsych 7?

**Comment by PerceptionCognitionLab** [comment]
This page totally works.  I used it to set up my csv save functions.  

https://kywch.github.io/jsPsych-in-Qualtrics/save-php/
